src/testflinger_device_connectors/fw_devices/LVFS/tests/fwupd_data.py

```diff
@@ -0,0 +1,387 @@
+""" fwupdmgr json outputs """
```

Comment:
The data here seems very specific, and I'm guessing we grabbed it from a fwupdmgr run to use as test data. Can you add a few details in the comments here about how it was extracted, just in case the format ever changes and we need to update the unit test data?